openquake/commonlib/calculators/base.py

```diff
@@ -332,3 +334,57 @@ def save_pik(self, result, **kw):
             with open(cache, 'w') as f:
                 cPickle.dump(self.risk_out, f)
         return self.risk_out
+
+
+# functions useful for the calculators ScenarioDamage and ScenarioRisk
+
+
+def get_gmfs(self):
+    """
+    :param self: a ScenarioDamage or ScenarioRisk selfulator
```

Comment:
Please check if the refactoring calc -> self produced any other words like selfulator